requirements-dev.txt

```diff
@@ -4,6 +4,7 @@
 #
 #    pip-compile --output-file requirements-dev.txt requirements-dev.in
 #
+appdirs==1.4.3            # via setuptools
```

Comment:
I removed the offending lines, reinstalled the whole virtualenv and it still works.